### ISSUE TITLE
Sample/Example Projects

### ISSUE THREAD

**Issue by isaacrg** [jagenjo/webglstudio.js#9]
Would it be possible for a few example projects to be accessible from this Github repo, or the website? I would love to play around with some samples, to get an understanding of what can be made.

**Comment by jagenjo** [comment]
you are right, there should be some examples in the default installation, but I dont want to fill the github with assets, so maybe I will add to litefileserver the option to import a packed folder so I can add examples and people can install it in his version.

**Comment by isaacrg** [comment]
In the short term, please could you put the examples on the web somewhere?

**Comment by jagenjo** [comment]
ok, I will next week

**Comment by isaacrg** [comment]
Brilliant, thanks

**Comment by timuric** [comment]
+1

**Comment by isaacrg** [comment]
@jagenjo Is there any progress on this?

**Comment by jagenjo** [comment]
The problem is that most of the examples use files that could have copyright issues (like the Lee face, or some cubemaps). Im trying to create new assets so they are free to share.

**Comment by magadan** [comment]
Hi Javi, for cubemaps its possible to use NOEMOTION hdr images that have more than 150 images available by Creative Commons License.

http://noemotionhdrs.net/

**Comment by isaacrg** [comment]
Are there any examples which are now without copyrighted assets?

**Comment by isaacrg** [comment]
Is there any progress on this

**Comment by jagenjo** [comment]
Sorry Isaac, going very slowly, finishing some important fixes in the litefileserver so it is easy to share examples.

**Comment by isaacrg** [comment]
Ok, cool
